source/_integrations/timer.markdown

```diff
@@ -16,7 +16,7 @@ When a timer finishes or gets canceled the corresponding events are fired. This
 
 
 ## Configuration
-The preferred way to configure timer helpers is via the user interface. To add one, go to Settings -> Automations & Scenes, select the "Helpers" tab and click the add button; next choose the “Timer” option.
+The preferred way to configure timer helpers is via the user interface at **Settings** -> **Devices & Services** -> **Helpers** and click the add button; next choose the **Timer** option.
```

Comment:
If we update these paths, it is preferable we link them using a My Home Assistant link.